src/scripts/convert-challenge-callout.ts

```diff
@@ -0,0 +1,171 @@
+import { createConfigUsingEnvVars } from '../util/create-config-using-envvars';
+import { AlkemioCliClient } from '../client/AlkemioCliClient';
+import { createLogger } from '../util/create-logger';
+import {
+  CalloutState,
+  CalloutType,
+  CreateAspectOnCalloutInput,
+  CreateCalloutOnCollaborationInput,
+  UpdateCalloutInput,
+  UpdateVisualInput,
+} from '@alkemio/client-lib';
+import { UpdateCalloutPublishInfoInput } from '../generated/graphql';
+
+const main = async () => {
+  await convertChallengeToCallout();
+};
+
+export const convertChallengeToCallout = async () => {
```

Comment:
Your code is hacky, but generic enough. Put `hubID`, `challengeIDs`, `publisherID`, `baseURL` as input params to `convertChallengeToCallout` and you have a workable solution. Then just have one more method calling `convertChallengeToCallout` with:

```typescript
  const hubID = 'un-sdgs';
  const challengeIDs = ['food'];
  const publisherID = 'neil-smyth-ae29597e-82e4-';
  const baseURL = 'https://demo.alkem.io';
```

and you have a workable solution that can be used in different scenarios

Reply:
Thanks, update is made...